freqtrade/freqtradebot.py

```diff
@@ -952,15 +959,13 @@ def _notify_sell(self, trade: Trade, order_type: str):
             'profit_percent': profit_percent,
             'sell_reason': trade.sell_reason,
             'open_date': trade.open_date,
-            'close_date': trade.close_date or datetime.utcnow()
+            'close_date': trade.close_date or datetime.utcnow(),
+            'stake_currency': self.config['stake_currency'],
         }
 
-        # For regular case, when the configuration exists
-        if 'stake_currency' in self.config and 'fiat_display_currency' in self.config:
-            stake_currency = self.config['stake_currency']
+        if 'fiat_display_currency' in self.config:
             fiat_currency = self.config['fiat_display_currency']
```

Comment:
I suspect we could remove this line too - it has no visible benefit (not even flake compliance due to line length)...

Reply:
Right, but this requires some changes into code handling those dict items in rpc/telegram and rpc/api-server, so I left it for the next PR to keep this one consisting of minor changes that do not change behavior.

Reply:
I think you misunderstood...
moving `self.config['fiat_display_currency']` into the `msg.update()` function won't require changes.

I've done the change i meant ... ;)